cluster/gce/util.sh

```diff
@@ -643,6 +643,15 @@ function kube-up {
     --type "${MASTER_DISK_TYPE}" \
     --size "${MASTER_DISK_SIZE}"
 
+  # Create disk for cluster registry if enabled
+  if [[ ${ENABLE_CLUSTER_REGISTRY} == true && -n ${CLUSTER_REGISTRY_DISK} ]]; then
```

Comment:
Nit: Quotes around `"${ENABLE_CLUSTER_REGISTRY}"` and `${CLUSTER_REGISTRY_DISK}`